<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F8P7PPD" failures="2" tests="2" name="Test(failed)" time="0.106" errors="0" timestamp="2022-06-24T14:51:19 IST">
  <testcase classname="TestNgStudy.TestFailexml1" name="p" time="0.018">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNgStudy.TestFailexml1.p(TestFailexml1.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- p -->
  <testcase classname="TestNgStudy.TestFaileXML" name="a" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNgStudy.TestFaileXML.a(TestFaileXML.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- a -->
</testsuite> <!-- Test(failed) -->
